lib/neo4j/active_node/persistence.rb

```diff
@@ -29,6 +29,14 @@ def save(*)
       end
     end
 
+    # Increments concurrently a numeric attribute by a centain amount
+    # @param [Symbol, String] name of the attribute to increment
+    # @param [Integer, Float] amount to increment
+    def concurrent_increment!(attribute, by = 1)
+      query_node = Neo4j::Session.query.match_nodes(n: neo_id)
+      increment_by_query! query_node, attribute, by
```

Comment:
I'm refactoring this in a branch, but I wanted to FYI you: When you have an object you can simply do `query_as(:n)`, so this method can just be:

```ruby
increment_by_query! query_as(:n), attribute, by
```

In my case the reason that's useful is that there are a limited number of places that `Neo4j::Session` is mentioned to reduce coupling

(Thanks again ;)